test/functional/api/v1/architectures_controller_test.rb

```diff
@@ -75,4 +75,12 @@ def user_one_as_manager
     end
     assert_response :success
   end
+
+  test "403 response contains missing permissions" do
```

Comment:
I thought the same but I didn't find a way how to easily add new action that would be authorized by some permission, note that I'm checking specific permission name in response so I'd have to create it and add it to access control list but there's no API for it. Stubbing wouldn't be easy and probably would result being very fragile.

that's why I took the simple controller and verify it on existing action